main.tex

```diff
@@ -450,11 +447,10 @@ \subsubsection{Defining functions that require quantities}
         enforce that the input units have the correct type.
         For example, we may want to require only length-type \texttt{Quantity}
         objects.
-        These requirements often lead to implementing repetitive code for
-        validating \texttt{Quantity} inputs.
-        \texttt{astropy.units} now provides a \python decorator,
-        \texttt{quantity\_input()}, that does this verification automatically.
-
+        \texttt{astropy.units} provides a tool called \texttt{quantity\_input()}
```

Comment:
tool reads to me like "utils" for @adrn, why cannot we be more precise?

Reply:
I want to avoid the word "decorator" which is very python-specific. Even a Ruby or R developer won't know what that is. So I first wrote "a wrapper for the function". That is a little more generic term, but then it's not much better than tool. Obviously, any programming tool is a tool. Emacs is a tools, python is a tool for research, ... I agree that "tool" is not the best word, but it's the best I could come up with that will e understood outside the Python community.

Reply:
but then "python decorator" is a googleable term

Reply:
It is. "Astropy" is a googable term and yet we still write a paper about it. Sorry for being snarky ;-) Any paper will clearly rely on the reader to know something already, but neither is it acceptable for papers to use lots of terms that are uncommon in the community (astronomy here). When I say "I process the image using a Hilbert filter", then the referee will rightly ask for a short explanation and maybe a reference, although Hilbert filter is a well defined and know  term in some community (but not astronomy). I think that "python decorator" is a term that needs explanation when used in an astronomy paper; I would not think that any explanation was needed if we were to submit this to the proceedings for Pycon (If they had any). 

So, I wonder, is it worth it to explain exactly what a decorator is just to use this word once or twice? I don't think so. Even in Python decorators are often called "syncatic sugar", they are just a short-hand form to write a function that calls the "decorated" function.

Reply:
To be honest I don't mind either way just get this out of the door asap. (I see and agree with your point, but then while the new version is more readable it doesn't have the same info content). And well, I believe the original need for the paper was for totally different reasons 😉